language/symbol-pool/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "symbol-pool"
```

Comment:
I'd name the crate `move-symbol-pool` ... maybe
My main concern is not making it clear it is designed for compilers (or other short lived programs) with its lack of ref counting and general leakyness